lib/main_common.pm

```diff
@@ -1496,6 +1496,7 @@ sub load_extra_tests_desktop {
 }
 
 sub load_extra_tests_textmode {
+    loadtest 'console/journalctl';
```

Comment:
If you don't mind please move it to the end of test modules. I guess by default we should assume that existing test modules are more known to people and we want to prevent that tests fail suddenly in a new module and former test modules are not executed anymore. If for testing purposes you want to just schedule e.g. boot and your new test module I suggest to use either `SCHEDULE` or `INCLUDE_MODULES` or `EXCLUDE_MODULES` from https://github.com/os-autoinst/os-autoinst/blob/master/doc/backend_vars.asciidoc, e.g. `INCLUDE_MODULES=boot/boot_to_desktop,console/journalctl` as parameter to openqa-clone-job.